README.md

```diff
@@ -60,42 +60,37 @@ You may also wish to visit Intel Media Server Studio [support page](https://soft
 Media SDK test and sample applications may require additional software packages (for example, X Server, Wayland, LibDRM, etc.) to be functional.
 
 # How to build
-## Requirements
- - Git* (with [LFS](https://git-lfs.github.com/) support)
- - Perl* v5.16+
- - Cmake* v2.8+
- - GCC* v4.8+
- - [LibVA](https://github.com/intel/libva)
 
 ## Build steps
 
-Get sources
+Get sources:
 ```sh
 git clone https://github.com/Intel-Media-SDK/MediaSDK msdk
 cd msdk
 ```
 
-Configure build with GCC (default) compiler:
+To configure and build Media SDK install cmake version 2.8.5 or later and run the following commands:
 ```sh
-perl tools/builder/build_mfx.pl --cmake=intel64.make.release
+mkdir build && cd build
+cmake ..
+make
+make install
 ```
-This will build MSDK binaries and MSDK samples.
+Media SDK depends on a number of packages which are identified and checked for the proper version during configuration stage. Please, make sure to install these packages to satisfy Media SDK requirements. After successful configuration 'make' will build Media SDK binaries and samples. The following cmake configuration options can be used to customize the build:
 
-If you want to configure build with CLang compiler use the following command:
-```sh
-perl tools/builder/build_mfx.pl --cmake=intel64.make.release.clang
-```
-
-Run build:
-```sh
-make -j8 -C __cmake/intel64.make.release
-```
+| Option | Values | Description |
+| ------ | ------ | ----------- |
+| API | master\|latest\|major.minor | Build mediasdk library with specified API. 'latest' will enable experimental features. 'master' will configure the most recent available published API (that's the default). |
 
 ## Enabling Instrumentation and Tracing Technology
 To enable the Instrumentation and Tracing Technology API you need either Intel® VTune™ Amplifier installed or to manually build an open source version. You can get ITT source files from [GitHub](https://github.com/01org/IntelSEAPI/tree/master/ittnotify) and build it on your own.
 
 **Please note** that auto detection of the Intel VTune Amplifier configuration is not supported. The next step is mandatory if you want to use this feature: set `$ITT_PATH` so `$ITT_PATH/include/ittnotify.h` and `$ITT_PATH/libittnotify64.a` will be valid paths. MSDK build system will automatically detect it.
 
+# Recommendations
+
+* In case of GCC compiler it is strongly recommended to use GCC version 6 or later since that's the first GCC version which has non-experimental support of C++11 being used in Media SDK.
+
```

Comment:
In addition we can add link to the infrastructure documentation https://github.com/Intel-Media-SDK/infrastructure#deploy-build-box-worker-buildbot
There is described needed packages and its versions which are 100% stable. I can modify infrastructure documentation with the needs of product documentation.

Reply:
You have a section in README now where you can add it. Let's do that in some of the next commit.

Reply:
Great, will do that.